protocol/tb.go

```diff
@@ -40,5 +40,5 @@ func (tb *TemporaryBinding) Serialize(strSig []byte) []byte {
 // the existed binding (TOFU).
 func (tb *TemporaryBinding) Verify(index, value []byte) bool {
 	return bytes.Equal(tb.Index, index) &&
-		(value != nil && bytes.Equal(tb.Value, value))
+		(value == nil || (value != nil && bytes.Equal(tb.Value, value)))
```

Comment:
When is value nil?  Also, if we do need this, it should be `(value == nil || bytes.Equal(tb.Value, value))`.